src/shell/dotnet/Shell/Fdc3/Fdc3StartupAction.cs

```diff
@@ -23,8 +23,22 @@ public Task InvokeAsync(StartupContext startupContext, Func<Task> next)
     {
         if (startupContext.ModuleInstance.Manifest.ModuleType == ModuleType.Web)
         {
-            startupContext.GetOrAddProperty<WebStartupProperties>()
+            var webProperties = startupContext.GetOrAddProperty<WebStartupProperties>();
+            webProperties
                 .ScriptProviders.Add(_ => new ValueTask<string>(ResourceReader.ReadResource(ResourceNames.Fdc3Bundle)));
+
+            webProperties
```

Comment:
This should be either added by the shell (because it's not FDC3-specific), or added under a different name, eg. `window.composeui.fdc3.instanceConfig`